### PR TITLE
Replace gate code with event Fixes #862

### LINKED ISSUE #862: Gate API Event ID adaptation
better description tbd

- [x] New endpoint get_events

- [x] adapt gate api used by app to work with event_id (not event code)

### PR DESCRIPTION
**Pull request by ArtyomKa**
Fixes #862 
### Proposed solution

Changed the usage of gate_code to event id in gate api. 

### Testing Done

Manual test of web gate interface. Mobile was not tested

### CODE REVIEW

File: routes/pages/gate_routes.js [hunk 1]
```diff
@@ -9,7 +9,8 @@ const Event = require('../../models/event').Event;
 router.get('/', userRole.isGateManager(), function (req, res) {
     Event.forge({event_id: req.user.currentEventId}).fetch().then(event => {
         return res.render('pages/gate', {
-            gate_code: event.attributes.gate_code
+            gate_code: event.attributes.gate_code,
```
Comment (NitayRabi) [inline routes/pages/gate_routes.js:12]: Why keep the `gate_code` here?

Comment (ArtyomKa) [reply routes/pages/gate_routes.js:12]: no special reason, just wasn't sure whether or not it used elsewhere.

Comment (NitayRabi) [reply routes/pages/gate_routes.js:12]: Ok! looks good IMO - I updated the branch and if it passes ill merge 👍

**Comment by ArtyomKa** [comment]
Thanks

**Pull request merged**